test/e2e/autoscale_test.go

```diff
@@ -253,6 +257,7 @@ func assertScaleDown(ctx *testContext) {
 		ctx.t.Fatalf("Waiting for Pod.List to have no non-Evicted pods of %q: %v", ctx.deploymentName, err)
 	}
 
+	ctx.logger.Info("Sleeping 10 seconds.")
```

Comment:
can you add a comment here as to why we need the sleep.

Reply:
this isnt new and honestly i have no idea why this is needed here.
i am simply adding logging to indicate to the user what is happening.

i can follow up in a future pr to investigate what the purpose of the sleep is for.